src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -980,6 +980,10 @@ void Convolution::createDescriptor(const std::vector<MemoryDescPtr>& inputDesc,
         memory::data_type bdt = outDnnlDesc.get_data_type();
 #else
         memory::data_type bdt = memory::data_type::f32;
+        // brdgmm_dw_conv supports only bia_type the same as src_type or dst_type
```

Comment:
Maybe add more comments here: 
/*   brdgmm_dw_conv has more perf gain on bf16/fp16 inference.
    brdgmm_dw_conv supports only bia_type the same as src_type or dst_type. 
dw convolution support in onednn 3.5. 
  BF16: 
    kernel     type                  |         brgdconv                            |    jit_uni_dw_convolution_fwd_t
    support impl type           |  native bf16 ISA without AMX    |      avx512_core_bf16 or  avx512_core
     bias dt                            |    oneof(src,dest)                        |      oneof(src, dest, f32)                         
  
  FP16: 
    kernel     type                  |         brgdconv                             |   brgemm_convolution_fwd_t
    impl type                         | native FP16 ISA without AMX     |     native FP16 ISA
     bias type                        |    oneof(src,dest)                          |      oneof(src, dest, f32)        
  */

Reply:
done